Client/Frontend/Components/FakespotErrorCardView.swift

```diff
@@ -0,0 +1,142 @@
+        stackView.addArrangedSubview(UIView())
+        stackView.axis = .vertical
+        stackView.distribution = .equalCentering
+        stackView.spacing = UX.iconStackViewSpacing
+    }
+
+    private lazy var infoContainerStackView: UIStackView = .build { stackView in
+        stackView.addArrangedSubview(self.iconStackView)
+        stackView.addArrangedSubview(self.labelContainerStackView)
+        stackView.axis = .horizontal
+        stackView.spacing = UX.horizontalStackViewSpacing
+    }
+
+    private lazy var labelContainerStackView: UIStackView = .build { stackView in
+        stackView.addArrangedSubview(self.titleLabel)
+        stackView.addArrangedSubview(self.descriptionLabel)
+        stackView.axis = .vertical
+        stackView.spacing = UX.verticalStackViewSpacing
+    }
+
+    private lazy var iconImageView: UIImageView = .build { imageView in
+        imageView.contentMode = .scaleAspectFit
+        // Set the icon image here
+        imageView.image = UIImage(named: StandardImageIdentifiers.Large.criticalFill)
+    }
+
+    private lazy var titleLabel: UILabel = .build { label in
+        label.textColor = .white
+        label.font = DefaultDynamicFontHelper.preferredBoldFont(
+            withTextStyle: .footnote,
+            size: UX.buttonFontSize)
+        label.setContentCompressionResistancePriority(.required, for: .vertical)
+        label.accessibilityIdentifier = AccessibilityIdentifiers.ErrorCard.title
+    }
+
+    private lazy var descriptionLabel: UILabel = .build { label in
+        label.textColor = .white
+        label.font = DefaultDynamicFontHelper.preferredFont(
+            withTextStyle: .footnote,
+            size: UX.buttonFontSize)
+        label.numberOfLines = 0
+        label.setContentCompressionResistancePriority(.required, for: .vertical)
+        label.accessibilityIdentifier = AccessibilityIdentifiers.ErrorCard.description
+    }
+
+    private lazy var primaryButton: ResizableButton = .build { button in
+        button.titleLabel?.font = DefaultDynamicFontHelper.preferredBoldFont(
+            withTextStyle: .callout,
+            size: UX.buttonFontSize)
+        button.layer.cornerRadius = UX.buttonCornerRadius
+        button.titleLabel?.textAlignment = .center
+        button.addTarget(self, action: #selector(self.primaryAction), for: .touchUpInside)
+        button.titleLabel?.adjustsFontForContentSizeCategory = true
+        button.contentEdgeInsets = UIEdgeInsets(top: UX.buttonVerticalInset,
+                                                left: UX.buttonHorizontalInset,
+                                                bottom: UX.buttonVerticalInset,
+                                                right: UX.buttonHorizontalInset)
+        button.accessibilityIdentifier = AccessibilityIdentifiers.ErrorCard.primaryAction
+    }
+
+    /// Custom init method to pass title, description text, action text
+    init(title: String, description: String, actionTitle: String) {
```

Comment:
We should use the default init (`init(frame:)`) and have a configure method where we pass a view model in. This way we can use `.build` when we use the card.

Reply:
Sure. It's the pattern I noticed for Components, I can make the change.